### PR TITLE
Reply to message method

### PR DESCRIPTION
**Pull request by mafineeek**
This pr cames with new method for message, for ez replying to messages :)

### CODE REVIEW

File: src/maps/Messages.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import type { Message as MessageI, SystemMessage } from 'revolt-api/types/Channe
 import type { Attachment } from 'revolt-api/types/Autumn';
 import type { Embed } from 'revolt-api/types/January';
 import type { Route } from '../api/routes';
+import { ulid } from 'ulid';
```
Comment (insertish) [inline src/maps/Messages.ts:5]: Useless import?

File: src/maps/Messages.ts [hunk 2]
```diff
@@ -126,6 +127,15 @@ export class Message {
...
         this.channel?.ack(this);
     }
+
+    /**
+     * Reply to Message
+     */
+
+    async reply(data: string | (Omit<Route<'POST', '/channels/id/messages'>["data"], 'nonce'> & { nonce?: string }), mention = true) {
+        let obj = typeof data === 'string' ? { content: data } : data;
+        this.channel?.sendMessage({ ...obj, replies: [{ id: this._id, mention }] })
```
Comment (insertish) [inline src/maps/Messages.ts:137]: Please either add a `return await` statement here, or remove `async` from the function and just add `return`.

**Review by insertish — CHANGES_REQUESTED** [review]
Looks great, just two issues that I've spotted.

File: src/maps/Messages.ts (hunk 2 above)
Comment (mafineeek) [reply src/maps/Messages.ts:137]: what it should return? or its supposed to be only `return`

Comment (insertish) [reply src/maps/Messages.ts:137]: either:
```js
async reply(..) {

return await this.channel.sendMessage
}
```
or:
```js
reply(..) {

return this.channel.sendMessage
}
```

**Comment by mafineeek** [comment]
ready to merge and publish ig

**Review by insertish — APPROVED** [review]

**Pull request merged**